### DISCUSSION TITLE
Autoremove Plus

### DISCUSSION THREAD

**Discussion by nobewayo** [lrstanley/deluge-remove-after#9] [General]
Isn't this already doing this? Just uncheck delete torrent and it will pause after a certain time..?

**Comment by lrstanley** [comment]
> Isn't this already doing this? Just uncheck delete torrent and it will pause after a certain time..?

I believe I tested the latest version of AutoremovePlus (v2) in the past, and it was non-functional -- it wasn't actually removing or pausing torrents, despite it being configured correctly. It would show in the logs that it ran, and that it deleted/paused torrents, but that wasn't actually reflected in Deluge.

it's possible this was due to how I was running deluge (docker w/ nfs). Haven't tried it since I wrote this though.
